src/vs/workbench/browser/media/part.css

```diff
@@ -30,9 +30,12 @@
 	padding-left: 12px;
 }
 
-.monaco-workbench > .part > .title > .title-label span {
+.monaco-workbench > .part > .title > .title-label h2 {
 	font-size: 11px;
 	cursor: default;
+	font-weight: normal;
```

Comment:
Why are these css changes needed? Because headings automatically have a margin and a font?

Reply:
Yes.